include/server.h

```diff
@@ -7,7 +7,7 @@
 
 typedef struct Server {
     unsigned int port;
-
+    int epollfd;
```

Comment:
Do not expose the implementation details in public headers.

Reply:
Instead, you can add a member such as `priv` which refers to "private" in the structure. In the implementation part, you can cast it with the proper type.